frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -147,13 +149,46 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({ data, loaded, ocsProp
   const [error, setError] = React.useState('');
   const [inProgress, setProgress] = React.useState(false);
   const [selectedNodesCnt, setSelectedNodesCnt] = React.useState(0);
+  const [isValidNodes, setValidNodes] = React.useState(true);
```

Comment:
yes, we are checking for invalid nodes count and based on that, updating `isValidNodes` state to true if no invalid nodes are present. Maybe renaming this to `areValidNodes` is better. @afreen23

Reply:
IMO it makes sense to name state on the context we are handling. So that if I come to back my code after a week could comprehend it.
Also down you are negating the `isValidNodes`  to show the component, so this makes sense to me to set the value to `true` at once.

This is just an enhancement for code readability purposes. This does not blocks PR from merging.